src/utils/isPlainObject.ts

```diff
@@ -2,7 +2,7 @@
  * @param {any} obj The object to inspect.
  * @returns {boolean} True if the argument appears to be a plain object.
  */
-export default function isPlainObject(obj) {
+export default function isPlainObject(obj: unknown) {
```

Comment:
A bunch of these functions are about comparing arbitrary objects so I've default to `unknown` everywhere. If the code made assumptions (such as using an indexer) then I changed it to use `any` instead.

I didn't think the `FixTypeLater` type was relevant here since there will never be a concrete type to change it to, though correct me if I'm wrong.

Reply:
I'm inclined to agree. We don't know what the type is at this point, and from the perspective of the code it doesn't really matter either.